src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -103,149 +73,107 @@
      * problematic if we allow multiple instructors for a single course.
      * TODO: make case insensitive
      */
+    public static final String COURSE_ID_FIELD_NAME = "a Course ID";
     public static final int COURSE_ID_MAX_LENGTH = 40;
-    public static final String COURSE_ID_ERROR_MESSAGE =
-            "\"%s\" is not acceptable to TEAMMATES as a Course ID because it %s. "
-            + "A Course ID can contain letters, numbers, fullstops, hyphens, underscores, and dollar signs. "
-            + "It cannot be longer than " + COURSE_ID_MAX_LENGTH + " characters. "
-            + "It cannot be empty or contain spaces.";
-    /*
-     * =======================================================================
-     * Field instructor permission role
-     */
-    public static final String INSTRUCTOR_ROLE_ERROR_MESSAGE =
-            "\"%s\" is not accepted to TEAMMATES as a role %s."
-            + "Role can be one of the following: "
-            + Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER + ", "
-            + Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_MANAGER + ", "
-            + Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_OBSERVER + ", "
-            + Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_TUTOR + ", "
-            + Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM + ", ";
-    public static final String INSTRUCTOR_ROLE_ERROR_REASON_NOT_MATCHING =
-            "it does not match the predifined roles";
 
-    /*
-     * =======================================================================
-     * Field: Feedback session start/end times
-     * Start time should be before end time.
-     * Only 1 hour increments allowed.
-     * TODO: allow smaller increments.
-     */
-    public static final String START_TIME_FIELD_NAME = "start time";
-    public static final String END_TIME_FIELD_NAME = "end time";
+    // TODO: allow smaller increments (currently only 1 hour increments allowed)
```

Comment:
Not your fault, but this TODO has nothing to do with the field name. Let's move it to the more relevant place.